Src/JsonDiffPatchDotNet/ItemMatch.cs

```diff
@@ -26,14 +26,29 @@ public virtual bool Match(JToken object1, JToken object2)
             return Match(object1, object2, ObjectHash);
```

Comment:
nit: Should we convert spaces to tabs in this file to avoid mixed indentation styles?

Reply:
Maybe just this file to keep the PR scoped.